clientconn_parsed_target_test.go

```diff
@@ -34,7 +34,7 @@ import (
 )
 
 func (s) TestParsedTarget_Success_WithoutCustomDialer(t *testing.T) {
-	defScheme := resolver.GetDefaultScheme()
+	defScheme := "passthrough"
```

Comment:
these tests use the `Dial` method, which we overrode to use the passthrough scheme, since the default scheme is now `dns`. I can instead have these tests use the `NewClient` method? Or I can also add additional tests for `Dial` vs `NewClient` methods

Reply:
Yes, I like the sound of making this test check both `Dial` and `NewClient`.

It would also be nice to have some tests for `NewClient` generally (at least creating a client and performing RPCs).